orbit-2-coroutines/src/main/java/com/babylon/orbit2/coroutines/CoroutineDslPlugin.kt

```diff
@@ -42,16 +42,24 @@ object CoroutineDslPlugin : OrbitDslPlugin {
     ): Flow<Any?> {
         return when (operator) {
             is TransformSuspend<*, *, *> -> flow.map {
+                if (operator.registerIdling) containerContext.settings.idlingRegistry.increment()
+
                 @Suppress("UNCHECKED_CAST")
                 with(operator as TransformSuspend<S, E, Any>) {
                     withContext(containerContext.backgroundDispatcher) {
                         createContext(it).block()
                     }
+                }.also {
+                    if (operator.registerIdling) containerContext.settings.idlingRegistry.decrement()
                 }
             }
             is TransformFlow<*, *, *> -> flow.flatMapConcat {
+                if (operator.registerIdling) containerContext.settings.idlingRegistry.increment()
+
                 with(operator as TransformFlow<S, E, Any>) {
                     createContext(it).block().flowOn(containerContext.backgroundDispatcher)
+                }.also {
+                    if (operator.registerIdling) containerContext.settings.idlingRegistry.decrement()
```

Comment:
I'm not sure this really works with finite flows or observables, have you tested it?

Basically you want this to increment when the observable/flow is subscribed to it and decrement when it's completed, rather than do this around the creation of it as that might not work as expected

I.e. we need something like 

``` kotlin
flow {
    if (operator.registerIdling) containerContext.settings.idlingRegistry.increment()
    emitAll(createContext(it).block().flowOn(containerContext.backgroundDispatcher))
    if (operator.registerIdling) containerContext.settings.idlingRegistry.decrement()
}
```